src/renderer/util/index.js

```diff
@@ -168,9 +168,10 @@ export const animatedScrollTo = function (element, to, duration, callback) {
 
   const animateScroll = function () {
     const now = +new Date()
-    const val = Math.floor(easeInOutQuad(now - animationStart, start, change, duration))
-
-    element.scrollTop = val
+    if (duration > 0) {
```

Comment:
@toSayNothing Could we handle this at the beginning because the duration didn't change?

https://github.com/marktext/marktext/blob/12fd124f98b9aec83c22558e7b2960a4317db143/src/renderer/util/index.js#L157

Reply:
I had push a new commit, cc? 😅
```js
  // Prevent animation on small steps or duration is 0
  if (duration === 0 || Math.abs(change) <= 6) {
    element.scrollTop = to
    return
  }
```